ClosedXML/Excel/Rows/XLRowCollection.cs

```diff
@@ -20,9 +20,7 @@ public void Add(int key, XLRow value)
         {
             if (key > MaxRowUsed) MaxRowUsed = key;
 
-            if (Deleted.ContainsKey(key))
-                Deleted.Remove(key);
-
+             Deleted.Remove(key);
```

Comment:
Remove can be called regardless whether dictionary contains something, no need for double lookups